### DISCUSSION TITLE
The mermaid script is always inserted, is this correct?

### DISCUSSION THREAD

**Discussion by ayaco-zg** [google/docsy#1588] [Q&A] [answered]
A mermaid Script tag is always inserted into pages that do not use mermaid.
I have specified [params.mermaid] enable=false in hugo.config, but it has no effect.
Is this behaviour in accordance with the specification?
Note that Hugo V0.113.0 and Docsy@0.7.0 are used.

Given that it says "With hugo version 0.93 or higher, support of Mermaid diagrams is automatically enabled **as soon as you use a mermaid code block on your page**." in the [This document](https://www.docsy.dev/docs/adding-content/diagrams-and-formulae/#diagrams-with-mermaid), I would imagine that this behaviour is not as expected.

**Comment by deining** [comment, marked as answer]
> A mermaid Script tag is always inserted into pages that do not use mermaid.

No. A correctly stated in the user guide, Mermaid script are only inserted for pages that actually contain mermaid code blocks.

> I have specified [params.mermaid] enable=false in hugo.config, but it has no effect.

Please remove the whole `[params.mermaid]` section from your config file. Since you use hugo V0113.0, it is no needed. Afterwards you will see mermaid scripts only on pages that actually use mermaid code blocks.

**Comment by ayaco-zg** [comment]
Thank you very much !
I did what you said and the script tag disappeared.

**Comment by curranjohn46** [comment]
Does mermaid work with docsy still? Mermaid blocks are not rendering. I have hugo version hugo v0.120.3-a4892a07b41b7b3f1f143140ee4ec0a9a5cf3970+extended
